packages/product-service/src/handlers/getProductById.ts

```diff
@@ -0,0 +1,64 @@
+import { APIGatewayProxyHandler } from "aws-lambda";
+import "source-map-support/register";
+
+import { getProductById } from "src/services";
+
+export const getProductByIdHandler: APIGatewayProxyHandler = async (
+  event,
+  _context
+) => {
+  const { productId } = event.pathParameters;
+  const headers = {
+    "Access-Control-Allow-Origin": "*",
+  };
+
+  try {
+    const product = await getProductById(productId);
+
+    if (product) {
+      const statusCode = 200;
+      return {
+        statusCode,
+        headers,
+        body: JSON.stringify(
+          {
+            message: "Success",
+            statusCode,
+            product,
+          },
+          null,
+          2
+        ),
+      };
```

Comment:
I'd advice creating some function so not to create this object from scratch every time. This would increase the readability.